src/sentry/middleware/integrations/tasks.py

```diff
@@ -0,0 +1,79 @@
+import logging
+from typing import Any, Dict, List
+
+import requests
+import sentry_sdk
+
+from sentry.models.outbox import ControlOutbox
+from sentry.silo.base import SiloMode
+from sentry.silo.client import RegionSiloClient
+from sentry.tasks.base import instrumented_task
+from sentry.types.region import get_region_by_name
+from sentry.utils import json
+
+logger = logging.getLogger(__name__)
+
+
+@instrumented_task(
+    name="sentry.middleware.integrations.tasks.convert_to_async_slack_response",
+    queue="integrations",
```

Comment:
```suggestion
    queue="integrations.control",
```
You need to use a queue that is consumed in control mode. This might be why the task was not firing.